sqlglot/dialects/dialect.py

```diff
@@ -220,6 +220,24 @@ class Dialect(metaclass=_Dialect):
     For example, such columns may be excluded from `SELECT *` queries.
     """
 
+    PREFER_CTE_ALIAS_COLUMN = False
```

Comment:
is this untrue in all dialects and only true in snowflake?

Reply:
I've only found it in Snowflake so far and tried Postgres, DuckDB, and BigQuery. @barakalon tried it in MySQL + SQLlite as well I think.

Reply:
Yeah, I tried MySQL, SQLite, Postgres, Trino, and Hive